mission/config/subconfigs/vehicle_respawn_info.hpp

```diff
@@ -813,438 +840,538 @@ class vehicles {
-				icon = VEHICLE_ICON_ARMOUR;
-				include[] = { { "m113","firesupport" } };
-				exclude[] = { { "Transport", "pavn" } };
-			};
-			class trucks {
-				name = "Trucks";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "truck","firesupport" } };
-				exclude[] = { { "pavn" } };
-			};
-		};
+		lockTeams[] = LOCKED_LIMITED_ARMOR;
 	};
 
-	class anzac {
-		name = "ANZAC Vehicles";
+	// 633rd //////////////////////////////////////////////////////////////////////////////
+
+	class 633rd_grnd_ambulance: parent_grnd_ambulance {
+		RESPAWN_LONG;
+		lockTeams[] = LOCKED_633RD;
+	};
+	class 633rd_air_transport {
+		name = "Transport Helicopters (633rd)";
 		RESPAWN_SHORT;
+		lockTeams[] = LOCKED_633RD;
 		class categories {
-			class dirtranger_t {
-				name = "Dirt Ranger";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "dirtranger", "transport" } };
-				exclude[] = { {"armed" } };
-			};
-			class dirtranger_p {
-				name = "Armed Dirt Ranger";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "dirtranger", "armed" } };
-				exclude[] = { {"transport" } };
+			class uh1 {
+				name = "UH-1";
+				icon = VEHICLE_ICON_HELO;
+				include[] = { {"uh1","transport","usaf" } };
+				exclude[] = { {"firesupport", "7thcav","usmc","arvn","army"} };
 			};
 		};
 	};
-
-	class usmc_light {
-		name = "USMC Wheeled Vehicles";
+	class 633rd_air_dustoff_medical {
+		name = "Dust-off and Medevac Helicopters (633rd)";
 		RESPAWN_SHORT;
+		lockTeams[] = LOCKED_633RD;
 		class categories {
-			class m54_t {
-				name = "M54";
-				icon = VEHICLE_ICON_TRUCK;
-				include[] = { { "m54","usmc" } };
+			class uh1 {
+				name = "UH-1 Medical";
+				icon = VEHICLE_ICON_HELO;
+				include[] = { { "uh1", "medical", "dustoff" } };
 				exclude[] = { { "pavn" } };
 			};
-			class m151a {
-				name = "M151A1";
-				icon = VEHICLE_ICON_CAR;
-				include[] = { { "m151a1", "usmc" } };
+			class ch47 {
+				name = "Ch-47 Medical";
+				icon = VEHICLE_ICON_HELO;
+				include[] = { { "ch47", "medical", "dustoff" } };
 				exclude[] = { { "pavn" } };
 			};
 		};
 	};
 
-	class usmc_heavy {
-		name = "USMC Armor Vehicles";
-		RESPAWN_SHORT;
-		class categories {
-			class m113 {
-				name = "APC";
-				icon = VEHICLE_ICON_ARMOUR;
-				include[] = { { "m113","armed","firesupport","apc" } };
-				exclude[] = { { "artillery","pavn" } };
-			}
-			class usmctank {
-				name = "Tanks";
-				icon = VEHICLE_ICON_TANK;
-				include[] = { { "tank","usmc" } };
-				exclude[] = { { "pavn","arvn" } };				
-			};
-		};
-	};
-
+	// 7th cav //////////////////////////////////////////////////////////////////////////
 
-	class air_scout {
-		name = "Scout air";
-		RESPAWN_SHORT;
-
-		class categories {
-			class cayuse {
-				name = "OH-6A";
-				icon = VEHICLE_ICON_HELO;
-				include[] = { {"oh6a","transport" } };
-				exclude[] = { {"firesupport","scout" } };
-			};
-		};
+	class 7thcav_grnd_patrol: parent_grnd_patrol {
+		RESPAWN_MEDIUM;
+		lockTeams[] = LOCKED_7THCAV;
 	};
```

Comment:
removed as didn't seem to be used, is just a repeat of the cayuse config (`air_transport_light`)